app/javascript/dashboard/routes/dashboard/search/components/SearchView.vue

```diff
@@ -0,0 +1,152 @@
+      <search-tabs :tabs="tabs" @tab-change="tab => (selectedTab = tab)" />
+    </header>
+    <div class="search-results">
+      <woot-loading-state v-if="uiFlags.isFetching" :message="'Searching'" />
+      <div v-else>
+        <div v-if="all.length">
+          <search-result-contacts-list
+            v-if="filterContacts"
+            :contacts="contacts"
+            :query="query"
+          />
+          <search-result-messages-list
+            v-if="filterMessages"
+            :messages="messages"
+            :query="query"
+          />
+          <search-result-conversations-list
+            v-if="filterConversations"
+            :conversations="conversations"
+            :query="query"
+          />
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import SearchHeader from './SearchHeader.vue';
+import SearchTabs from './SearchTabs.vue';
+import SearchResultConversationsList from './SearchResultConversationsList.vue';
+import SearchResultMessagesList from './SearchResultMessagesList.vue';
+import SearchResultContactsList from './SearchResultContactsList.vue';
+import { mapGetters } from 'vuex';
+
+export default {
+  components: {
+    SearchHeader,
+    SearchTabs,
+    SearchResultContactsList,
+    SearchResultConversationsList,
+    SearchResultMessagesList,
+  },
+  data() {
+    return {
+      selectedTab: 'all',
+      query: '',
+    };
+  },
+  computed: {
+    ...mapGetters({
+      fullSearchRecords: 'conversationSearch/getFullSearchRecords',
+      uiFlags: 'conversationSearch/getUIFlags',
+    }),
+    contacts() {
+      if (this.fullSearchRecords.contacts) {
+        return this.fullSearchRecords.contacts.map(contact => ({
+          ...contact,
+          type: 'contact',
+        }));
+      }
+      return [];
+    },
+    conversations() {
+      if (this.fullSearchRecords.conversations) {
+        return this.fullSearchRecords.conversations.map(conversation => ({
+          ...conversation,
+          type: 'conversation',
+        }));
+      }
+      return [];
+    },
+    messages() {
+      if (this.fullSearchRecords.messages) {
+        return this.fullSearchRecords.messages.map(message => ({
+          ...message,
+          type: 'message',
+        }));
+      }
+      return [];
+    },
+    all() {
+      return [...this.contacts, ...this.conversations, ...this.messages];
+    },
+    filterContacts() {
+      return this.selectedTab === 'contacts' || this.selectedTab === 'all';
+    },
+    filterConversations() {
+      return this.selectedTab === 'conversations' || this.selectedTab === 'all';
+    },
+    filterMessages() {
+      return this.selectedTab === 'messages' || this.selectedTab === 'all';
+    },
+    tabs() {
+      return [
+        {
+          key: 'all',
+          name: this.$t('SEARCH.TABS.ALL'),
+          count:
+            this.contacts.length +
+            this.conversations.length +
+            this.messages.length,
```

Comment:
Can we create a separate computed property name like `totalCount` or `totalSearchCount`?
```suggestion
```

Reply:
Fixed